legacy/models.py

```diff
@@ -191,11 +188,32 @@ def get_fields(self):
         blood_book_result_fields_to_dict.append("allergen")
         return blood_book_result_fields_to_dict
 
+    def is_significant(self):
+        if self.precipitin:
+            if not self.precipitin == self.NEGATIVE:
+                return True
+        if self.rast and self.rast >= 2:
+            return True
+        if self.kul:
+            # sometimes the user puts in < 0.1, < 0.35, > 100
+            if "<" in self.kul:
+                kul = self.kul.strip(" <>")
+                if float(kul) <= 0.35:
```

Comment:
Perhaps worth handling kul == o.l ?

Reply:
maybe what does `o.l` mean?

Reply:
e.g. any non float castable string not involving the characters`<>`

Reply:
Its a great point. Resolved. (Although I think when we speak to the user next we should talk about whether this could be a float field)